storage/mysql/storage.sql

```diff
@@ -80,10 +80,10 @@ CREATE TABLE IF NOT EXISTS LeafData(
   LeafIdentityHash     VARBINARY(255) NOT NULL,
   -- This is the data stored in the leaf for example in CT it contains a DER encoded
   -- X.509 certificate but is application dependent
-  LeafValue            BLOB NOT NULL,
+  LeafValue            LONGBLOB NOT NULL,
   -- This is extra data that the application can associate with the leaf should it wish to.
   -- This data is not included in signing and hashing.
-  ExtraData            BLOB,
+  ExtraData            LONGBLOB,
```

Comment:
I think we should probably allow for it as we don't know what applications will really need. Clients won't be able to submit huge blobs unless the database server is configured for it.